esp-hal/src/i2c/master/mod.rs

```diff
@@ -16,34 +16,9 @@
 //!
 //! The I2C driver implements a number of third-party traits, with the
 //! intention of making the HAL inter-compatible with various device drivers
-//! from the community, including the [`embedded-hal`].
+//! from the community, including the [embedded-hal].
 //!
-//! ## Examples
```

Comment:
Instead of deleting this, could you move it to the docs for the I2c driver itself? Essentially so this appears on this page: https://docs.esp-rs.org/esp-hal/esp-hal/0.23.1/esp32c6/esp_hal/i2c/master/struct.I2c.html